submissions/ArthurGorbenko/Memory Pair Game/js/script.js

```diff
@@ -0,0 +1,128 @@
+const valuesOfBackPos = [
+  "20px 0px",
+  "20px -180px",
+  "20px -350px",
+  "20px -525px",
+  "-190px 0px",
+  "-390px 0px",
+  "-600px 0px",
+  "-810px 0px"
+];
+
+let cards = document.getElementsByClassName("card");
+const container = document.querySelector(".container");
+
+for (let i = 0; i < cards.length; i++) {
```

Comment:
Don't forget use function for this operation))

Reply:
If i understood you clearly, you mean change `for` loop for some function ? As i know there is only one way to loop throw HTMLCollection and it is `for` loop.Please,correct me if i'm made a mistake or did't understand you ;)

Reply:
I mean don`t write some code in global scope It's bad practice 
Create some wrapper function and place this code there 
It will make your code easy to understanding because in future when you have big
project that will be harder to read and support

Reply:
Thank's a lot for detailed answers,now it is more clear for me ;)